starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,53 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" href="./css/main.css"/>
+
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+  <div>
+<h1 class ="ok">Apple Pies </h1>
+  </div>
+<div>
+<p class="oui"> This was my grandmother's apple pie recipe. I have never seen another one <br>
+quite like it. It will always be my favorite and has won me several first place <br>
+prizes in local competitions. I hope it becomes one of your favorites as well! </p>
+<div class ="new">
```

Comment:
It may be easier for you to just put your img tag without div, also instead of having a "trait" div, try the <hr> tag :)

Reply:
"<hr>" tag

Reply:
< hr > tag*